app.js

```diff
@@ -203,8 +190,12 @@ exports.init = function (isA11yTest = false, a11yTestSession = {}, ftValue) {
             req.session.language = 'en';
         }
 
-        if (req.query && req.query.locale && config.languages.includes(req.query.locale)) {
-            req.session.language = req.query.locale;
+        if (req.query) {
+            if (req.query.lng && config.languages.includes(req.query.lng)) {
+                req.session.language = req.query.lng;
+            } else if (req.query.locale && config.languages.includes(req.query.locale)) {
+                req.session.language = req.query.locale;
+            }
         }
```

Comment:
You only need this for PCQ:
```
        if (req.query && req.query.lng && config.languages.includes(req.query.lng)) {
            req.session.language = req.query.lng;
        }
```
(I kept `locale` as the link to Probate and Caveats from GOV.UK use that and asking them to change it is a big PITA)